bio/cnvkit/target/meta.yaml

```
@@ -0,0 +1,13 @@
+name: "cnvkit target"
+url: https://github.com/etal/cnvkit
+description: >
+ Prepare a BED file of baited regions for use with CNVkit. See documentation https://cnvkit.readthedocs.io/en/stable/pipeline.html
+input:
+  - bed: bed file
+  - annotate: gene annotation databases, e.g. RefSeq or Ensembl, as a “flat” format (can be found at UCSC)
```

Is this file mandatory or optional? Because right now it is not being provided.
```suggestion
  - annotate: gene annotation databases, e.g. RefSeq or Ensembl, as a “flat” format (can be found at UCSC) (optional)
```

---

Its optional!